lib/Runtime/Library/ScriptFunction.cpp

```diff
@@ -221,6 +244,20 @@ using namespace Js;
         this->GetScriptFunctionType()->ChangeEntryPoint(entryPointInfo, entryPoint, isAsmJS);
```

Comment:
We lost a bit here because it moved to a different file. No functional difference but we can avoid some duplication and prevent further merge conflicts when this payload goes to Windows: [https://devdiv.visualstudio.com/DevDiv/Chakra/_git/ChakraCore/pullrequest/136749?_a=overview](https://devdiv.visualstudio.com/DevDiv/Chakra/_git/ChakraCore/pullrequest/136749?_a=overview)